brats/utils.py

```diff
@@ -1,4 +1,31 @@
 import torch
+import numpy as np
+
+
+def is_binary(input: torch.Tensor) -> bool:
+    """
+    Check whether the input contains only 0s and/or 1s
+    Args:
+        input: Tensor to check
+    Returns:
+        bool: True if contains only 0s and/or 1s, False otherwise
+
+    """
+    zero_and_one = torch.tensor([0., 1.], device=input.device)
+    return all([value in zero_and_one for value in torch.unique(input)])
+
+
+def has_only_zeros(input: np.ndarray) -> bool:
+    """
+    Check whether the input contains only 0s.
+    Args:
+        input: Array to check
+
+    Returns:
+        bool: True if contains only 0s, False otherwise
+
+    """
+    return not np.any(input != 0)
```

Comment:
I think it would be clearer to just write `return np.all(input==0)` :P

Reply:
I an alternative yerba drinker now - such pathetically obvious solutions don't satisfy me anymore